### PR TITLE
Reword and fix a typo

### PR DESCRIPTION
**Pull request by tagrudev**
I am just changing the README and fixing a typo in the `run_length` file, also I am adding an `answers` folder for a better structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Instructions
 
 1. Clone this repository
-2. Checkout to a branch which has your name. Example: Deyan Dobrinov => `deyan-dobrinov`
-3. Solve questions [1; N] available in the root directory
+2. Create a new branch, use your first and last name. Example: Deyan Dobrinov => `deyan-dobrinov`
```
Comment (dobrinov) [inline README.md:4]: My previous explanation was probably not good enough. I am usually creating a branch for the candidate who I am going to interview in advance, so the branch exists. In this branch I am usually committing few symbolic links to question which I would like to include in the interview. Basically the idea here is to see the interviewee perform some basic operations with Git. Nothing else.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+3. Solve the `/questions` and add them in `/answers`
```
Comment (dobrinov) [inline README.md:5]: The questions directory would contain a "pool" of questions. The idea is that I will be creating symbolic links in the root directory to a selected set of questions.

**Review by dobrinov — COMMENTED** [review]
Could you provide more details around the idea behind the "answers" directory?

**Comment by tagrudev** [comment]
I am closing that as I did not get your `symlink` idea from the provided documentation.

**Pull request closed without merging**